ui/ducks/send/send.js

```diff
@@ -1054,10 +1054,9 @@ const slice = createSlice({
         recipient.error = null;
         recipient.warning = null;
       } else {
-        const isSendingToken =
-          asset.type === ASSET_TYPES.TOKEN ||
-          asset.type === ASSET_TYPES.COLLECTIBLE;
```

Comment:
@adonesky1 
I removed this piece of code as the `isSendingToken` is not necessary here for my PR. In order to make sending ETH possible,`|| asset.type ===  ASSET_TYPES.NATIVE` should be added or `isSendingToken` removed from here and down there. 
ASSET_TYPES looks like this:
`export const ASSET_TYPES = {
  NATIVE: 'NATIVE',
  TOKEN: 'TOKEN',
  COLLECTIBLE: 'COLLECTIBLE',
};
`
In this case, if another condition was added, `isSendingToken` would always be true.  
Agreed with Dan about this one.
// cc @danjm

Reply:
Looks good to me!

Reply:
Thanks for pointing this out though!